tests/ignite/contrib/handlers/test_base_logger.py

```diff
@@ -117,19 +117,23 @@ def update_fn(engine, batch):
 
         mock_log_handler = MagicMock()
 
-        logger.attach(trainer, log_handler=mock_log_handler, event_name=event)
+        logger.attach(trainer, log_handler=mock_log_handler, event_name=event, **kwargs)
 
         trainer.run(data, max_epochs=n_epochs)
 
         if isinstance(event, EventsList):
             events = [e for e in event]
         else:
             events = [event]
-        calls = [call(trainer, logger, e) for e in events]
-        mock_log_handler.assert_has_calls(calls)
-        assert mock_log_handler.call_count == n_calls
 
-    _test(Events.ITERATION_STARTED, len(data) * n_epochs)
+        if len(kwargs) > 0:
```

Comment:
That's not right, here you're asserting only if `len(kwargs) > 0`, please edit the test